src/commands/case_management.py

```diff
@@ -589,7 +591,7 @@ async def cmd_case_management_sub(ctx: Context):
             last_author=ctx.user.nickname,
             author=rescue.quotes[quote_id].author,
             created_at=rescue.quotes[quote_id].created_at,
-            updated_at=datetime.now(timezone.utc),
+            updated_at=pendulum.now(pendulum.tz.UTC),
```

Comment:
[style] We should be consistent, is it `pendulum.now()` or `pendulum.now(pendulum.tz.UTC)`.
I believe the two forms are consistent, since pendulum datetimes are always TZ aware, defaulting to the current locale.
Logically, either don't specify a timezone or always use UTC. Since all pendulum datetimes are tz-aware, it appears to be a stylistic choice.